test/fs-admin.test.js

```diff
@@ -74,6 +70,8 @@ describe('fs-admin', function () {
   })
 
   describe('makeTree', () => {
+    if (process.platform === 'linux') return
```

Comment:
I won't hold up the PR for this change, but I've opened https://github.com/atom/fs-admin/issues/16 to think about how to better organize these tests now we have some improved Linux support.